Jenkinsfile

```diff
@@ -2,50 +2,6 @@
 
 library("govuk")
 
-REPOSITORY = 'govuk_schemas'
-
 node {
-
-  try {
-    stage('Checkout') {
-      checkout scm
-    }
-
-    stage('Clean') {
-      govuk.cleanupGit()
-      govuk.mergeMasterBranch()
-    }
-
-    stage("Set up content schema dependency") {
-      govuk.contentSchemaDependency()
-    }
-
-    stage('Bundle') {
-      echo 'Bundling'
-      sh("bundle install --path ${JENKINS_HOME}/bundles/${JOB_NAME}")
-    }
-
-    stage('Linter') {
-      govuk.lintRuby()
-    }
-
-    stage('Tests') {
-      govuk.setEnvar('RAILS_ENV', 'test')
-      govuk.runTests('spec')
-    }
-
-    if(env.BRANCH_NAME == "master") {
-      stage('Publish Gem') {
-        govuk.publishGem(REPOSITORY, env.BRANCH_NAME)
-      }
-    }
-
-  } catch (e) {
-    currentBuild.result = 'FAILED'
-    step([$class: 'Mailer',
-          notifyEveryUnstableBuild: true,
-          recipients: 'govuk-ci-notifications@digital.cabinet-office.gov.uk',
-          sendToIndividuals: true])
-    throw e
-  }
+  govuk.buildProject()
```

Comment:
It'd be good to put `rubyLintDiff: false` so it lints the full code base, hopefully this doesn't blow up the build. Sorry I shouldn't have commented on this with my previous comment too to save you two changes.